libraries/AP_HAL_ChibiOS/UARTDriver.cpp

```diff
@@ -28,6 +30,8 @@
 #include <AP_Common/ExpandingString.h>
 #include "Scheduler.h"
 #include "hwdef/common/stm32_util.h"
+// MAVLink is included to use the MAV_POWER flags for the USB power
+#include <GCS_MAVLink/GCS_MAVLink.h>
```

Comment:
isn't including mavlink in the hal a no-no? Can we move the flags somewhere else?

Reply:
This is not a highly functional include. Also it's only included in the src file not header. I believe we do this in gpio.cpp as well.